src/main/io/osd.c

```diff
@@ -2982,7 +2980,7 @@ static void osdShowStats(void)
         displayWrite(osdDisplayPort, statValuesX, top++, buff);
 
         int32_t totalDistance = getTotalTravelDistance();
-        if (totalDistance > 0) {
+        if (STATE(GPS_FIX)) {
```

Comment:
Not sure this is a good idea. You may have landed w/o GPS fix, yet the efficincy data might still be be more-less accurate.

Reply:
That's a good point, but max speed, max distance and traveled distance use the same check. Maybe change both to `feature(FEATURE_GPS)`?

Reply:
Sounds good!

Reply:
Done.